From b038c3452667ed17ddb0e791cd7bdc7f8774ac29 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sat, 11 May 2019 20:35:20 +0800
Subject: [PATCH] Allow us to pass in PYTHON_SITE_PACKAGES

The python binary used when building for nativesdk doesn't give us the
correct path here so we need to be able to specify it ourselves.

Upstream-Status: Inappropriate [config]
Signed-off-by: Paul Eggleton <paul.eggleton@linux.intel.com>

Rebase to 2.9.9

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 configure.ac | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index ca911f3..3bbd654 100644
--- a/configure.ac
+++ b/configure.ac
@@ -808,7 +808,8 @@ dnl
 
 PYTHON_VERSION=
 PYTHON_INCLUDES=
-PYTHON_SITE_PACKAGES=
+# Allow this to be set externally
+#PYTHON_SITE_PACKAGES=
 PYTHON_TESTS=
 pythondir=
 if test "$with_python" != "no" ; then
-- 
2.7.4

